django_plastic_tickets/templates/plastic_tickets/ticket_view.html

```diff
@@ -42,8 +42,8 @@ <h3> # Ticket {{ ticket.id }}</h3>
                     </a>
                 </td>
                 <td>{{ config.count }}</td>
-                <td>{{ config.material_type }}</td>
-                <td>{{ config.color }}</td>
+                <td>{{ config.get_material_type_name }}</td>
+                <td>{{ config.get_color_name }}</td>
```

Comment:
Maybe you also want to display the same info as in the ticket email (labels of possible materials) here.
Also consider displaying this only to the staff.